src/utils.ts

```diff
@@ -116,19 +116,15 @@ function addToSet(
 /**
  * Clean up an array to harden against CSPP
  *
- * @param array - The array to be cleaned.
- * @returns The cleaned version of the array
+ * @param {T[]} array - The array to be cleaned.
+ * @returns {null[]} The cleaned version of the array
  */
-function cleanArray(array: any[]): any[] {
+function cleanArray<T>(array: T[]): null[] {
   for (let index = 0; index < array.length; index++) {
-    const isPropertyExist = objectHasOwnProperty(array, index);
-
-    if (!isPropertyExist) {
-      array[index] = null;
-    }
+    array[index] = null;
```

Comment:
I don't think we need to validate index with hasOwnProperty since we're iterating the length of the array anyway!